libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3437,10 +3439,12 @@ void GCS_MAVLINK::handle_named_value(const mavlink_message_t &msg) const
 
 void GCS_MAVLINK::handle_system_time_message(const mavlink_message_t &msg)
```

Comment:
better to wrap the whole function and never call it from `case MAVLINK_MSG_ID_SYSTEM_TIME` too just like how handle_optical_flow() is

Reply:
I went backwards and forwards on this one.  I went this way because I thought things other than RTC might want to use that message.

Changed it as you suggested.